README.md

```diff
@@ -55,7 +55,8 @@
 - 💜 [Vue.js](https://github.com/inertiajs/inertia/tree/master/packages/inertia-vue)
 - 💜 [React](https://github.com/inertiajs/inertia/tree/master/packages/inertia-react)
 - 💜 [Svelte](https://github.com/inertiajs/inertia/tree/master/packages/inertia-svelte)
-- [Mithril](https://github.com/tbreuss/inertia-mithril)
+- [Mithril](https://github.com/maicol07/inertia-mithril)
+- [Mithril (up to Inertia 0.1)](https://github.com/tbreuss/inertia-mithril)
```

Comment:
```suggestion
```

You can remove this one 👍

Reply:
:thinking: I don't know if my adapter works with oldest version, but if the list only allows latest Inertia versions, then I can remove it.

Reply:
The older Inertia versions are not supposed to be used any longer - if someone is starting a project and is looking for a Mithril adapter, they'll use the latest versions. So yes you can remove it 👍

Reply:
Done!